raster/identify-raster-cell/src/main/java/com/esri/samples/identify_raster_cell/IdentifyRasterCellSample.java

```diff
@@ -0,0 +1,220 @@
+import javafx.geometry.Point2D;
+import javafx.scene.Scene;
+import javafx.scene.control.Alert;
+import javafx.scene.input.MouseButton;
+import javafx.scene.input.MouseEvent;
+import javafx.scene.layout.StackPane;
+import javafx.stage.Stage;
+import java.io.File;
+import java.util.List;
+import java.util.concurrent.ExecutionException;
+
+import com.esri.arcgisruntime.concurrent.ListenableFuture;
+import com.esri.arcgisruntime.geometry.Point;
+import com.esri.arcgisruntime.layers.RasterLayer;
+import com.esri.arcgisruntime.loadable.LoadStatus;
+import com.esri.arcgisruntime.mapping.ArcGISMap;
+import com.esri.arcgisruntime.mapping.Basemap;
+import com.esri.arcgisruntime.mapping.GeoElement;
+import com.esri.arcgisruntime.mapping.view.Callout;
+import com.esri.arcgisruntime.mapping.view.IdentifyLayerResult;
+import com.esri.arcgisruntime.mapping.view.MapView;
+import com.esri.arcgisruntime.raster.Raster;
+import com.esri.arcgisruntime.raster.RasterCell;
+
+public class IdentifyRasterCellSample extends Application {
+
+    private boolean calloutLocked = false;
+    private Callout callout;
+    private MapView mapView;
+    private RasterLayer rasterLayer;
+
+    @Override
+    public void start(Stage stage) {
+
+        try {
+            // create stack pane and application scene
+            StackPane stackPane = new StackPane();
+            Scene scene = new Scene(stackPane);
+
+            // set title, size, and add scene to stage
+            stage.setTitle("Identify Raster Cell Sample");
+            stage.setWidth(800);
+            stage.setHeight(700);
+            stage.setScene(scene);
+            stage.show();
+
+            // create a map view
+            mapView = new MapView();
+
+            // create a Map with an oceans basemap
+            ArcGISMap map = new ArcGISMap(Basemap.Type.OCEANS, -33.9, 18.6, 9);
+
+            // add the map to a map view
+            mapView.setMap(map);
+
+            // create a raster from a local raster file
+            Raster raster = new Raster(new File(System.getProperty("data.dir"), "./samples-data/SA_EVI_8Day_03May20/SA_EVI_8Day_03May20.tif").getAbsolutePath());
+
+            // create a raster layer
+            rasterLayer = new RasterLayer(raster);
+
+            // add the raster as an operational layer
+            map.getOperationalLayers().add(rasterLayer);
+
+            // set viewpoint on the raster
+            rasterLayer.addDoneLoadingListener(() -> {
+                if (map.getLoadStatus() == LoadStatus.LOADED) {
+                    mapView.setViewpointGeometryAsync(rasterLayer.getFullExtent(), 150);
+                } else {
+                    Alert alert = new Alert(Alert.AlertType.ERROR, "Raster Layer Failed to Load!");
+                    alert.show();
+                }
+            });
+
+            // get a handle on the callout
+            callout = mapView.getCallout();
+            callout.setMouseTransparent(true);
```

Comment:
It basically means that hovering over the callout makes the app act as if we're hovering over the map. In other words, the callout is transparent to the mouse interaction. 
What it does is basically allow the on-the-fly identification to keep working if we move the mouse over the callout. Without it the app would stop identifying because we're technically hovering on the callout, and not on the map.
I'll add a better comment!

Reply:
ahhh ok I get it now, thanks for explaining!

Reply:
I've added a comment that hopefully explains this now. Let me know what you think?